cni/cmd/istio-cni/main.go

```diff
@@ -29,8 +29,12 @@ import (
 )
 
 func main() {
+	os.Exit(runPlugin())
```

Comment:
Nit - can `runPlugin` just return `err` and we have a tiny `if err return 1 else return 0` here? Would rather keep RC codes next to the OS exit, they aren't relevant to the rest of the code.

Reply:
+1